GPL/DemanglerGnu/src/demangler_gnu/c/cp-demangle.c

```diff
@@ -93,7 +93,7 @@ This file was changed on July 4th, 2019.
 
    IN_GLIBCPP_V3
       If defined, this file defines only __cxa_demangle() and
-      __gcclibcxx_demangle_callback(), and no other publically visible
+      __gcclibcxx_demangle_callback(), and no other publicly visible
```

Comment:
This is an open source file we have copied into our project.  Our goal is to change this file (and the others in this directory) as little as possible.   This change needs to be removed.

Reply:
Fixed